blacksauce/src/styles/grid/_footerGrid.scss

```diff
@@ -0,0 +1,65 @@
+//general grid for the social media
+.grid--socialMedia {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+}
+
+//icons for social media
+.facebookIcon {
+    grid-column: 1/2;
+}
+
+.instagramIcon {
+    grid-column: 2/3;
+}
+
+.twitterIcon {
+    grid-column: 3/4;
+}
+
+@media screen and (min-width:800px) {
+
+    // at the designated media query, change the display of the footer into a grid with two columns and three rows
+    footer {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-column-gap: 1em;
+        grid-template-rows: 3em 2fr 1fr;
+
+        h2,
```

Comment:
Remember to update these styles to use classnames and not the base elements from my code.